python_modules/dagster/dagster/core/asset_defs/assets.py

```diff
@@ -180,8 +184,11 @@ def from_graph(
             node_def=graph_def,
             asset_deps=transformed_internal_asset_deps or None,
             partitions_def=check.opt_inst_param(
-                partitions_def, "partitions_def", PartitionsDefinition
+                partitions_def,
+                "partitions_def",
+                PartitionsDefinition,
             ),
+            group_names_by_key={AssetKey(graph_def.name): group_name} if group_name else None,
```

Comment:
note to self: there's some code required here to ensure that all assets defined get assigned to this group_name. right now only a single asset with the graph name will get assigned to the group.